src/overlays/actors/ovl_En_Ru2/z_en_ru2.c

```diff
@@ -195,7 +224,7 @@ s32 func_80AF27D0(EnRu2* this, PlayState* play, u16 cueId, s32 cueChannel) {
     return false;
 }
 
-s32 func_80AF281C(EnRu2* this, PlayState* play, u16 cueId, s32 cueChannel) {
+s32 EnRu2_CheckCueIsNot(EnRu2* this, PlayState* play, u16 cueId, s32 cueChannel) {
```

Comment:
Hmm... I do think it's intended to be the inverse of `EnRu2_CheckCueIs`, but it isn't correct in the case where `cue == NULL`. I would tag it with a `@bug` comment, but maybe fig can weigh in as not to long ago we discussed this same sort of null check logic flaw.

Reply:
My read is that it's intended to be used with some cutscene data, which is why they don't want it NULL, but they want it to not match the cue ID here (as opposed to the other function where it should match).
I'll try renaming these two functions again. See what you think.